apps/studio/components/interfaces/Home/ProjectList/ProjectCard.tsx

```diff
@@ -26,7 +26,7 @@ const ProjectCard = ({
   const { name, ref: projectRef } = project
   const desc = `${project.cloud_provider} | ${project.region}`
 
-  const isBranchingEnabled = project.preview_branch_refs.length > 0
+  const isBranchingEnabled = project.preview_branch_refs?.length > 0
```

Comment:
[note]: I just want to prevent errors from being thrown before redirecting to a new page